ci/find-current-version.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Helper script to return the current lts or feature version based on
+# the branch and tags of Zeek's repository.
+#
+# * lts is the highest sorting release/x.0 branch which contains a tag
+#   of the form form '^v{x}.0.0$'.
+#
+# * feature is the highest sorting release/x.[^0]+ branch that does
+#   that contains a 'v{x}.{y}.0' tag
+#
+set -euo pipefail -x
+
+REMOTE=${REMOTE:-origin}
+MAIN_BRANCH=${MAIN_BRANCH:-refs/remotes/${REMOTE}/master}
+
+function usage() {
+    echo "Usage $0 <lts|feature>" >&2
+    exit 1
+}
+
+if [ $# -ne 1 ]; then
+    usage
+fi
+
+if [ "${1}" = "lts" ]; then
+    PATTERN=".* refs/remotes/${REMOTE}/release/[0-9]+\.0\$"
```

Comment:
Just curious why you match on remotes here — because we wouldn't use this on repos where the release branch only exists locally, not pushed yet? I think that's okay, just wanted to flag.

Reply:
Mainly because Cirrus clones the repo with single-branch and we then fetch the remote release heads as a follow-up:
https://github.com/zeek/zeek/pull/2766/files#diff-62587956f943bb2503db7bc6dd27d0d888074a1c0ecaab3f570ad611aff0f7bbR520

We could fetch them directly into `refs/heads` locally (or otherwise check out all release branches locally), but it seemed more explicit to indicate determination of feature/lts depends on the remote state of the repo (minimally in CI).